documentation/README.md

```diff
@@ -0,0 +1,29 @@
+# 📖 `dotnet-monitor` documentation
+
+`dotnet-monitor` is a tool that makes it easier to get access to diagnostics information in a dotnet process.
+
+When running a dotnet application differences in diverse local and production environments can make collecting diagnostics artifacts (e.g., logs, traces, process dumps) challenging. `dotnet-monitor` aims to simplify the process by exposing a consistent HTTP API regardless of where your application is run.
+
+## Table of contents
+
+- [Setup](./setup.md)
+- [Getting Started](#)
+    - [Running on a local machine](#)
+    - [Running in Docker](#)
+    - [Running in a kubernetes cluster](#)
+- [API Endpoints](#)
+    - [OpenAPI document](./openapi.json)
+    - [`/processes`](#)
+    - [`/dump`](#)
+    - [`/gcdump`](#)
+    - [`/trace`](#)
+    - [`/metrics`](#)
+    - [`/logs`](#)
+- [Configuration](#)
+    - [JSON Schema](./schema.json)
+- [Authentication](#)
```

Comment:
I think these should be "Windows" and "API Key". No need to repeat Auth each time and the API Key authentication is key based, not token based.